INSTALL_GUIDE_WINDOWS.md

```diff
@@ -200,7 +200,7 @@ AWS CloudWatch is required to build the AWS CDI SDK, and is provided in [AWS SDK
    - Place the **libfabric** and **libfabric_new** folders at the same directory level as the **aws-cdi-sdk** folder.
    - In the **libfabric_new** folder, rename the **libfabric.vcxproj**, **libfabric_new.vcxproj.filters** and **libfabric_new.vcxproj.user** files to use **libfabric_new** instead of **libfabric**. This is done for several reasons. The AWS CDI SDK solution file uses them as a project source and linking to the generated libraries. It also provides a clear identification of the libfabric versions while using the AWS CDI SDK solution in Visual Studio.
 2. Follow the Windows instructions to install [Network Direct SPI](https://github.com/ofiwg/libfabric#windows-instructions). Extract the header files from the downloaded NetworkDirect_DDK.zip and copy them from **NetDirect\include** to **libfabric_new\prov\netdir\NetDirect**.
-3. Clone the Windows RDMA repo linked at [Windows RDMA - TODO](https://TODO/). Place the **rdmawin** folder at the same directory level as the **aws-cdi-sdk** folder.
+3. Clone the Windows RDMA repo linked at [Windows EFA](https://github.com/aws/efawin/). Place the **rdmawin** folder at the same directory level as the **aws-cdi-sdk** folder.
```

Comment:
With this github package, the name of the repository is "efawin". So the text should read something like:

Clone the Windows EFA repo .... Place the **efawin** ...